src/jquery.data-remote.js

```diff
@@ -177,7 +196,7 @@
         var method = _options.oneAndDone ? 'one' : 'on';
 
         // bind to specific event type
-        $element[method](_options.eventType, function(evt) {
+        $element[method](_options.eventType, debounce(function(evt) {
```

Comment:
this is actually going to debounce the callback on every event, which is not what we want. We only want to debounce on events like key(up|down|press), scroll, etc.

Maybe we should also add an option to allow overriding the wait time as well? You can just add a default of 500 ms